res/controllers/midi-components-0.0.js

```diff
@@ -481,8 +481,7 @@
                 if (this.max === Component.prototype.max) {
                     this.max = (1 << 14) - 1;
                 }
-                value = (value << 7) + (this._firstLSB ? this._firstLSB : 0);
-                this.input(channel, control, value, status, group);
+                this.input(channel, control, (value << 7) + (this._firstLSB ? this._firstLSB : 0), status, group);
             }
             this.MSB = value;
```

Comment:
A quick word on the fix:
This only occurred when receiving the first value because this branch is only executed when receiving the first value. `this.MSB` is only supposed to contain the value of the most significant byte, but the branch erroneously set it contain the entire value. That meant if the MSB was received first everything would work out for the first message, however, when the LSB was received afterwards it would take the bogus MSB value set here resulting in huge bogus values. Once the second MSB was received, everything would work out as usual again. The fix now ensures that `this.MSB` gets the actual MSB value instead of the "pre-scaled" one that we use in the branch.

Reply:
LGTM